ompi/class/ompi_seq_tracker.h

```diff
@@ -34,37 +34,33 @@ typedef struct ompi_seq_tracker_range_t ompi_seq_tracker_range_t;
 
 OMPI_DECLSPEC OBJ_CLASS_DECLARATION(ompi_seq_tracker_range_t);
 
-struct ompi_seq_tracker_t{
-    opal_list_t seq_ids;         /**< list of seqs id's that have been seen */
-    ompi_seq_tracker_range_t* seq_ids_current; /**< a pointer to the last place we were in the list */
-
+struct ompi_seq_tracker_t {
+    opal_list_t seq_ids; /**< list of seqs id's that have been seen */
+    ompi_seq_tracker_range_t
```

Comment:
I know this is the tool, but rather than split this way, we should really use "line before" Doxygen style rather than moving this to two lines.

Reply:
Agreed. I strongly prefer line-before anyway. The results are cleaner in general and easier for tools like clang-format to produce clean output.

Reply:
I fixed this locally. Will look for other instances that should be fixed.